.travis.yml

```diff
@@ -260,91 +260,86 @@ jobs:
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3"
       script: ./tools/travis/nightly.sh split_sticky.sh
       name: sticky - hadoop 2.8
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3"
       script: ./tools/travis/nightly.sh split_checkpoints.sh
       name: checkpoints - hadoop 2.8
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3"
       script: ./tools/travis/nightly.sh split_container.sh
       name: container - hadoop 2.8
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3"
       script: ./tools/travis/nightly.sh split_heavy.sh
       name: heavy - hadoop 2.8
       # E2E profiles - Scala 2.12
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -De2e-metrics -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -De2e-metrics"
       script: ./tools/travis/nightly.sh split_misc.sh
       name: misc - scala 2.12
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12"
       script: ./tools/travis/nightly.sh split_ha.sh
       name: ha - scala 2.12
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12"
       script: ./tools/travis/nightly.sh split_sticky.sh
       name: sticky - scala 2.12
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12"
       script: ./tools/travis/nightly.sh split_checkpoints.sh
       name: checkpoints - scala 2.12
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12"
       script: ./tools/travis/nightly.sh split_container.sh
       name: container - scala 2.12
     - if: type = cron
-      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12 -Dinclude-kinesis"
+      env: PROFILE="-Dinclude-hadoop -Dhadoop.version=2.8.3 -Dscala-2.12"
       script: ./tools/travis/nightly.sh split_heavy.sh
       name: heavy - scala 2.12
       # E2E profiles - Hadoop-free
     - if: type = cron
-      env: PROFILE="-De2e-metrics -Dinclude-kinesis"
+      env: PROFILE="-De2e-metrics"
       script: ./tools/travis/nightly.sh split_misc_hadoopfree.sh
       name: misc
     - if: type = cron
-      env: PROFILE="-Dinclude-kinesis"
```

Comment:
set this to an empty string instead; otherwise the PROFILE value of the above job will be used. (the travis.yml is funky like that.........)